sg_covid_impact/sic.py

```diff
@@ -1,43 +1,59 @@
 import requests
 import os
 import logging
-import sg_covid_impact
 import re
+import json
 import pandas as pd
 
+# from toolz.curried import pipe
+
+import sg_covid_impact
+
 project_dir = sg_covid_impact.project_dir
 
+_SIC_OUTPUT_FILE = f"{project_dir}/data/raw/sic_2007.xls"
+_SIC_LOOKUP_FILE = f"{project_dir}/data/raw/lookup.json"
+
+
 def url():
-    return sg_covid_impact.config['fetch_urls']['sic_taxonomy'] 
+    return sg_covid_impact.config["fetch_urls"]["sic_taxonomy"]
 
-def extract_sic_code_description(table, var_name, var_name_tidy):
+
+def extract_sic_code_description(table, var_name):
     """Extracts codes and descriptions from SIC table
     Args:
         table (pandas.DataFrame): summary of the SIC taxonomy
         var_name (str): level of SIC we want to extract a lookup for
-        var_name_tidy (str): name for the variable code
     Returns:
         A lookup between the variable codes and their description
     """
     loc = list(table.columns).index(var_name)  # Find the location of class
     select = table.iloc[:, [loc, loc + 1]].dropna()  # Extract variable & description
     select.columns = [var_name, "description"]
 
-    select[var_name_tidy] = [re.sub(r"\.", "", str(x)) for x in select[var_name]]
-    name_lookup = select.set_index(var_name_tidy)["description"].to_dict()
+    select[var_name] = [re.sub(r"\.", "", str(x)) for x in select[var_name]]
+    name_lookup = select.set_index(var_name)["description"].to_dict()
     return name_lookup
```

Comment:
might as well avoid a variable assignment and having to think about what to name the return value:
 `return select.set_index(var_name)["description"].to_dict()`